core-examples/pom.xml

```diff
@@ -35,6 +35,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-handler-proxy</artifactId>
+      <version>4.1.0.CR7</version>
+      <scope>runtime</scope>
```

Comment:
We had the netty-handler-proxy as optional dependency of vertx-core, so I added the module with scope runtime for the examples (since they compile without it), this is no longer necessary since it is a direct dependency in vertx-core now, i will remove the dependency entryy